proto/ibc/applications/transfer/v1/events.proto

```diff
@@ -12,12 +12,19 @@ message EventTransfer {
     string receiver = 2;
 }
 
+// EventAcknowledgementSuccess is a typed event emitted on packet acknowledgement success
+message EventAcknowledgementSuccess {
```

Comment:
these can go in `channel`. We have a acknowledgement success/error type defined in channel so other applications may want to use this event. It's usage would make it easier on relayers as well